<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>760</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="dispGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>显示参数</string>
   </property>
   <widget class="QComboBox" name="cbDispMode">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>画中画  1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>画中画  2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>叠加     1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>叠加     2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>显示模式</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMirror">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>左右镜像</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbKeepRatio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保持长宽比</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnPhoto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QWidget" name="visible" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>256</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="visibleGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>243</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>可见光相机参数</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>对比度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelContrast">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="sliderContrast">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>50</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderBrightness">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>90</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBrightness">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="sliderSaturate">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>130</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>饱和度</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSaturate">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAWB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>自动白平衡</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>830</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="swirGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>241</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>短波相机参数</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>对比度</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGain">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sliderGain">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHighgain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>设置探测器高增益</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbNonuniform">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>开启非均匀性校正</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbIntegral">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>开启自动帧积分时间调整</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧周期(ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帧积分时间(ms)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdjustOnsize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>现场修正校正系数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCycle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editIntegral">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetCycle">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetIntegral">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbHistogram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>直方图均衡</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSmooth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>开启多帧平滑</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>存储路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>690</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desktop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
